lib/fog/ovirt/compute.rb

```diff
@@ -50,73 +50,53 @@ def respond_to_missing?(method_name, include_private = false)
         end
       end
 
-      require "fog/ovirt/compute/v3"
       require "fog/ovirt/compute/v4"
 
       def self.new(options = {})
         super(options)
-
-        # rubocop:disable Style/ConditionalAssignment
-        if options[:api_version] == "v4"
-          @client = Fog::Ovirt::Compute::V4.new(options)
-        else
-          @client = Fog::Ovirt::Compute::V3.new(options)
-        end
-        # rubocop:enable Style/ConditionalAssignment
+        @client = Fog::Ovirt::Compute::V4.new(options)
       end
 
-      # rubocop:disable Style/MethodMissingSuper, Style/MissingRespondToMissing
       def method_missing(symbol, *args)
         @client.__send__(symbol, *args)
+        super
       end
 
       def respond_to?(symbol, include_all = false)
         @client.respond_to?(symbol, include_all)
       end
-      # rubocop:enable Style/MethodMissingSuper, Style/MissingRespondToMissing
+      # rubocop:enable
 
       class Mock
         def initialize(options = {})
-          if options[:api_version] == "v4"
-            Fog::Ovirt::Compute::V4::Mock.include Fog::Ovirt::Compute::Collections
-            @client = Fog::Ovirt::Compute::V4::Mock.new(options)
-          else
-            Fog::Ovirt::Compute::V3::Mock.include Fog::Ovirt::Compute::Collections
-            @client = Fog::Ovirt::Compute::V3::Mock.new(options)
-          end
+          Fog::Ovirt::Compute::V4::Mock.include Fog::Ovirt::Compute::Collections
+          @client = Fog::Ovirt::Compute::V4::Mock.new(options)
         end
 
-        # rubocop:disable Style/MethodMissingSuper, Style/MissingRespondToMissing
         def method_missing(symbol, *args)
           @client.__send__(symbol, *args)
+          super
```

Comment:
Why `super` here?

Reply:
without the super, I'm keep getting rubocop errors: 
lib/fog/ovirt/compute.rb:60:7: C: Style/MethodMissingSuper: When using method_missing, fall back on super.
      def method_missing(symbol, *args) ...
      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/fog/ovirt/compute.rb:90:9: C: Style/MethodMissingSuper: When using method_missing, fall back on super.
        def method_missing(symbol, *args)